Added documentation about SSH in the --remote flag of deploy and destroy commands (#470)
okteto/docs: src/content/reference/cli.mdx

| _--var_                                     |   (list)   | Set a pipeline variable (can be set more than once)                                                                                                            |
| _--wait_                                    |   (bool)   | Wait until the pipeline finishes (defaults to false)                                                                                                           |
| _--no-bash_                                 |   (bool)   | Execute the command using the container's default shell instead of bash (defaults to false)                                                                    |

> | _--remote_                                  |   (bool)   | Run the deploy commands in your Okteto cluster. Okteto will connect your local SSH agent with the cluster so you so can for example clone any private repository as part of your manifest execution as long as your local agent has the proper ssh key configured                                                                                                                 |

Reviewer: ```suggestion
| _--remote_                                  |   (bool)   | Run the deploy commands in your Okteto cluster. Okteto will connect your local SSH agent with the cluster. For example, this lets you clone any private repository as part of your manifest execution as long as your local agent has the proper ssh key configured                                                                                                             |
```

| _--wait_                                 |  (bool)  | Wait until the development environment is destroyed (defaults to `false`)                                                                               |
| _--no-bash_                              |  (bool)  | Execute the command using the container's default shell instead of bash (defaults to false)                                                             |
| _--all_                                  |  (bool)  | Destroy everything (defaults to `false`)                                                                                                                |

> | _--remote_                               |  (bool)  | Run the destroy commands in your Okteto cluster. Okteto will connect your local SSH agent with the cluster so you so can for example clone any private repository as part of your manifest execution as long as your local agent has the proper ssh key configured                                                                                                         |

Reviewer: ```suggestion
| _--remote_                               |  (bool)  | Run the deploy commands in your Okteto cluster. Okteto will connect your local SSH agent with the cluster. For example, this lets you clone any private repository as part of your manifest execution as long as your local agent has the proper ssh key configured                                                                                                          |
```
